entity/src/advisory.rs

```diff
@@ -20,13 +27,37 @@ pub struct Model {
     pub title: Option<String>,
 }
 
+#[ComplexObject]
+impl Model {
+    async fn organization(&self, ctx: &Context<'_>) -> Result<organization::Model> {
+        let db: &Arc<db::Database> = ctx.data::<Arc<db::Database>>()?;
+        if let Some(found) = self
+            .find_related(organization::Entity)
+            .one(&db.connection(&db::Transactional::None))
+            .await?
+        {
+            Ok(found)
+        } else {
+            Err(Error::new("Organization not found"))
+        }
+    }
+
+    async fn vulnerabilities(&self, ctx: &Context<'_>) -> Result<Vec<vulnerability::Model>> {
+        let db: &Arc<db::Database> = ctx.data::<Arc<db::Database>>()?;
+        Ok(self
+            .find_related(vulnerability::Entity)
+            .all(&db.connection(&db::Transactional::None))
+            .await?)
+    }
+}
+
 #[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]
 pub enum Relation {
     #[sea_orm(
         belongs_to = "super::organization::Entity"
         from = "Column::IssuerId"
         to = "super::organization::Column::Id")]
-    Issuer,
+    Organization,
```

Comment:
Would still prefer to not rename this relationship.

Reply:
That was a left over, thanks. Done.